src/ai/backend/manager/cli/image_impl.py

```diff
@@ -118,6 +118,28 @@ async def forget_image(cli_ctx, canonical_or_alias, architecture):
                     ImageAlias(canonical_or_alias),
                 ],
             )
+            image_row.status = ImageStatus.DELETED
+            await session.flush()
```

Comment:
It feels awkward that the same logic is implemented separately in CLI and GraphQL.
Is it difficult to at least wrap them up as common functions?

Reply:
It's okay if you just define it as a follow-up work.

Reply:
I think it would be good to add it as a method of ImageRow, such as `ImageRow.delete()` or `ImageRow.mark_as_deleted()`.